functions/activate-batch-auto-background.js

```diff
@@ -78,6 +88,28 @@ async function handle(_, db, contract) {
     let result = await db.query('batches', 'batch', batch)
     const existingAddresses = result.Items
 
+    let addressBadluckCount = {};
+    for(let i = 0; i < existingAddresses.length; i+=MAX_CONCURRENCY) {
+      await Promise.all(existingAddresses.slice(i, i+MAX_CONCURRENCY).map(async a => {
+        let account = {
+          address: a.address,
+          inventory: '[]',
+          fragments: 0,
+          badLuckCount: 0,
+          discord: "",
+        }
+    
+        const fetchedAccount = (await db.get('accounts',{ address: a.address })).Item
+    
+        if(fetchedAccount !== undefined) {
+          account = fetchedAccount
+        }
+
+        addressBadluckCount[a.address] = account;
```

Comment:
Since this dictionary is for badLuckCounts, we should just store bad luck counts and not the whole account in here

Reply:
I think here can also check if `badLuckCount` exists on the fetched account otherwise initialise to 0